kinto-remote-settings/tests/signer/test_events.py

```diff
@@ -51,7 +36,7 @@ def get_app_settings(cls, extras=None):
         settings["event_listeners"] = "ks"
         settings[
             "event_listeners.ks.use"
-        ] = "kinto_remote_settings.signer.tests.test_events"
+        ] = "kinto_remote_settings.signer.private.mock_listener"
```

Comment:
nitpick: In kinto we have a `testing` submodule. I would be in favor of using the same name convention, and maybe it makes sense to move it at the root of the `kinto_remote_settings` package, since it's not fully related to signer, but resource events in general (`kinto_remote_settings.testing.mock_listener`)

Reply:
776a435e5e5b591c13f28bddb5a80cc51ca3a7fb